Add radio button option to select (#7752)
keystonejs/keystone: .changeset/metal-cars-breathe.md

---
'@keystone-6/core': minor
---

> Add 'radio' button display mode option to select field.

Reviewer: ```suggestion
Adds `ui.displayMode: 'radio'` as an option for the select field.
```
